### PR TITLE
feat: add basic error tracking and fixes user not found issue

### LINKED ISSUE #87: API Key creation failing
Upon trying to create a new API key using the createkey button, it still says "No Keys".

### LINKED ISSUE #92: add error tracking to API and Frontend

### PR DESCRIPTION
**Pull request by hugomrdias**
todo:

- [x] change sentry version/release to git commit

closes #92
closes #87

### CODE REVIEW

File: site/src/utils/utils.js [hunk 1]
```diff
@@ -8,17 +8,20 @@ export async function notFound(event) {
...
- * @param {() => Promise<T | void>} fn
+ * @param {(ctx : import("../bindings").RouteContext) => Promise<T | void>} fn
  * @param {string} label
+ * @param {import("../bindings").RouteContext} ctx
  * @returns {Promise<T | void>}
  */
-export async function timed(fn, label) {
-  console.log(`START: ${label}`)
-  console.time(`END: ${label}`)
+export async function timed(fn, label, ctx) {
```
Comment (alanshaw) [inline site/src/utils/utils.js:16]: This is no longer timing anything. Is it possible to log the time spent with sentry?

File: README.md [hunk 2]
```diff
@@ -53,6 +54,17 @@ npx simple-git-hooks
 
 ## `site` Setup
 
+### Local env vars
+
+Inside the `site` folder create a file called `.env.local` with the following content.
+
+```ini
+SENTRY_TOKEN=<sentry user auth token>
+SENTRY_UPLOAD=false
```
Comment (alanshaw) [inline README.md:63]:
```suggestion
SENTRY_TOKEN=<sentry user auth token>
SENTRY_UPLOAD=false # Enable/disable sending logs to Sentry
```

File: README.md [hunk 3]
```diff
@@ -53,6 +54,17 @@ npx simple-git-hooks
 
 ## `site` Setup
 
+### Local env vars
+
+Inside the `site` folder create a file called `.env.local` with the following content.
```
Comment (alanshaw) [inline README.md:59]: Why are these in an env file now and not `wrangler.toml` or an uploaded cloudflare secret like the rest?

File: site/src/utils/utils.js (hunk 1 above)
Comment (hugomrdias) [reply site/src/utils/utils.js:16]: there no timing inside workers, date.now console timing etc dont work

File: README.md (hunk 2 above)
Comment (hugomrdias) [reply README.md:63]: this is just for source/sourcemaps uploading, capturing still work even with this `false`

File: README.md (hunk 3 above)
Comment (hugomrdias) [reply README.md:59]: these vars are used in the webpack.config.js, this file doesnt know about CF secrets

File: README.md (hunk 2 above)
Comment (alanshaw) [reply README.md]: Ok, can we put this as the comment then - not obvious!

**Pull request merged**